.eslintrc.js

```diff
@@ -32,9 +32,6 @@ module.exports = {
     "max-len": "off",
     "comma-dangle": "off",
     "linebreak-style": "off",
-
-    // Things we might care about if we could automate them.
-    "curly": "off", // curly braces even single line blocks, --fix does something weird.
```

Comment:
Hoping whatever this was referencing in the past is now fixed, if @shans has opinion please comment.

Reply:
The problem was the eslint 'fixed' this by putting braces in weird places.  If you use clang-format you can automate the cleanup.

For example
```
  if (foo)
     return true;
```

would be rewritten as 
```
   if (foo)
     {return true;}
```

clang-format fixes that automatically